OWML.Launcher/App.cs

```diff
@@ -117,30 +111,11 @@ private bool IsMadeForSameOwmlMajorVersion(IModManifest manifest)
                    owmlVersionSplit[1] == modVersionSplit[1];
         }
 
-        private void ListenForOutput()
-        {
-            _listener.OnOutput += OnOutput;
-            _listener.Start();
-        }
-
-        private void OnOutput(string s)
-        {
-            var lines = s.Split(new[] { Environment.NewLine }, StringSplitOptions.RemoveEmptyEntries);
-            foreach (var line in lines)
-            {
-                _writer.WriteLine(line);
-                if (line.EndsWith(Constants.QuitKeyPhrase))
-                {
-                    ExitConsole();
-                }
-            }
-        }
-
         private bool HasVrMod(IList<IModData> mods)
         {
             var vrMod = mods.FirstOrDefault(x => x.Config.RequireVR && x.Config.Enabled);
             var hasVrMod = vrMod != null;
-            _writer.WriteLine(hasVrMod ? $"{vrMod.Manifest.UniqueName} requires VR." : "No mods require VR.");
+            _writer.WriteLine(MessageType.Log, hasVrMod ? $"{vrMod.Manifest.UniqueName} requires VR." : "No mods require VR.");
```

Comment:
I know it's not your code, but maybe we should output full list of mods that require VR 🤔